scripts/setup-ts-execution.js

```diff
@@ -1,6 +1,10 @@
 const path = require(`path`);
+const babel = require('@babel/core');
+const os = require('os');
 const root = path.dirname(__dirname);
 
+process.env.BABEL_CACHE_PATH = path.join(os.tmpdir(), 'babel', `.babel.${babel.version}.${babel.getEnv()}.json`);
```

Comment:
I wonder if that could lead to subtle issues if multiple clones of the repository use different caches. What would you think of keeping an ignored `.babel/cache` directory at the root of the repo?

Reply:
Maybe `<root>/.cache/babel-register` then?

Reply:
I think multiple clones should not result in issues, because babel computes hash from the source code prior to storing it into the cache. So one cache for multiple repos should generally work even better theoretically

Reply:
It actually stores physical path to the source file and compiled contents corresponding to that file, so one cache should be fine

Reply:
I see, good for me then, thanks for checking 👍